aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConflictHandlerResult.java

```diff
@@ -22,20 +22,20 @@
 public enum DataStoreConflictHandlerResult {
 
     /**
-     * Conflict is handled by discarding the local (client-side) changes, preferring whatever
+     * Conflict handled by discarding the local (client-side) changes, preferring whatever
      * was on the server.
      */
-    DISCARD,
+    APPLY_REMOTE,
 
     /**
-     * Conflict is handled by keeping the local (client-side) changes, discarding whatever
-     * was on the server.
+     * The conflict handled by retrying the local change which attempts to persist
+     * it again on the backend.
      */
-    KEEP,
+    RETRY_LOCAL,
 
     /**
-     * Conflict is handled by return a new `Model` instance that should used
+     * Conflict handled by returning a new `Model` instance that should be used
      * instead of the local and remote changes.
```

Comment:
The description here is not as good as what you wrote in the PR description. To that point, your PR description was actually better than most of the existing JavaDoc in the code. I wonder if you can find places to copy your PR description documentation into the JavaDoc? Please feel more than free to remove/overwrite anything that I had.